tests/tasks/enable_epel.yml

```diff
@@ -20,6 +20,19 @@
       until: __epel_status is success
       retries: 6
       delay: 10
+      when: ansible_distribution_major_version != '7'
+
+    - name: Create EPEL-7
```

Comment:
Let's avoid creating another task, for reusability, and because in this case we must use different `register` variable names, otherwise on EL 8 `__epel_status` variable will contain `skipped`.

```
    - name: Create EPEL {{ ansible_distribution_major_version }}
      vars:
        __epel_7:
          "https://dl.fedoraproject.org/pub/archive/epel/7/x86_64/\
          Packages/e/epel-release-7-14.noarch.rpm"
        __epel: https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{
            ansible_distribution_major_version }}.noarch.rpm
        __epel_url: "{{
          __epel_7 if ansible_distribution_major_version == '7'
          else __epel }}"
      command:
        cmd: rpm -iv {{ __epel_url }}
        # noqa command-instead-of-module
        creates: /etc/yum.repos.d/epel.repo
      register: __epel_status
      # sometimes the rpm download returns a 403 - I think it is when too
      # many parallel jobs attempt the download from the same controller in
      # too short a period of time, so the epel server throttles additional
      # downloads - use a retry here to mitigate
      until: __epel_status is success
      retries: 6
      delay: 10
```